app/Http/Controllers/Licenses/LicensesController.php

```diff
@@ -289,4 +290,99 @@ public function getClone($licenseId = null)
         ->with('item', $license)
         ->with('maintained_list', $maintained_list);
     }
+
+    /**
+     * Exports Licenses to CSV
+     *
+     * @author [G. Martinez]
+     * @since [v6.3]
+     * @return StreamedResponse
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function getExportLicensesCsv()
+    {
+        $this->authorize('view', License::class);
+        \Debugbar::disable();
+
+        $response = new StreamedResponse(function () {
+            // Open output stream
+            $handle = fopen('php://output', 'w');
+
+            License::with('company','manufacturer','category', 'supplier', 'adminuser')
```

Comment:
I don't see any company scoping in here.

Reply:
I think I pushed this  before your Company scope fix 😅 --anywho its added and tested. 👍